docker-compose-prod.yml

```diff
@@ -20,18 +20,18 @@ services:
       - database
 
   database:
-    image: mysql:5.7
-    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+    image: mysql:8-oracle
+    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci', '--socket=/tmp/mysql.sock']
     environment:
       - MYSQL_ROOT_PASSWORD=ccm_root_pw
       - MYSQL_HOST=ccm_db_prod
-      - MYSQL_PORT=3306
+      - MYSQL_TCP_PORT=3306
       - MYSQL_DATABASE=ccm
       - MYSQL_USER=admin
       - MYSQL_PASSWORD=admin
-    entrypoint: ['docker-entrypoint.sh', '--default-authentication-plugin=mysql_native_password']
     ports:
       - 5400:3306
     volumes:
-      - ./.mysql_data_prod:/var/lib/mysql:delegated
+      - ./.mysql_data_prod/mysql:/var/lib/mysql
+      - ./mysql:/docker-entrypoint-initdb.d:ro
```

Comment:
This line shouldn't be necessary for CCM -- at least not yet. This is to modify the permissions for the user so they can create new databases, which is necessary for Django applications with tests (e.g. Placement Exams or Canvas App Explorer).

Reply:
Right, this is to run init scripts in MySQL which this project doesn't have yet, but might need in the future.

Reply:
I saw a warning about change the password to sha2 something, without the line 36 things ran fine, but To remove that warning I kept it.

Reply:
I removed it now and I don't see the warning that I saw. Strange!